docs/content/concepts/partitions-schedules-sensors/asset-sensors.mdx

```diff
@@ -15,15 +17,17 @@ Asset sensors allow you to instigate runs when materializations occur.
 | <PyObject object="AssetSensorDefinition"  />               | A special sensor definition class for asset sensors. You almost never want to use initialize this class directly. Instead, you should use the <PyObject object="asset_sensor" decorator /> which returns a <PyObject object="AssetSensorDefinition"  />                                                                                                                 |
-| <PyObject object="freshness_policy_sensor" decorator />    | The decorator used to define a freshness policy sensor. The decorated function is an evaluation function that takes in a <PyObject object="FreshnessPolicySensorEvaluationContext" />. The decorator returns a <PyObject object="FreshnessPolicySensorDefinition" />                                                                                                    |
+| <PyObject object="freshness_policy_sensor" decorator />    | The decorator used to define a freshness policy sensor. The decorated function is an evaluation function that takes in a `FreshnessPolicySensorContext`. The decorator returns a <PyObject object="FreshnessPolicySensorDefinition" />                                                                                                                                  |
 | <PyObject object="FreshnessPolicySensorDefinition"  />     | A special sensor definition class for freshness policy sensors. You almost never want to use initialize this class directly. Instead, you should use the <PyObject object="freshness_policy_sensor" decorator /> which returns a <PyObject object="FreshnessPolicySensorDefinition"  />                                                                                 |
 | <PyObject object="multi_asset_sensor" decorator />         | The decorator used to define an asset sensor that can monitor multiple assets at a time. The decorated function is an evaluation function that takes in a <PyObject object="MultiAssetSensorEvaluationContext" /> which has methods to fetch materialization events for the monitored assets. The decorator returns an <PyObject object="MultiAssetSensorDefinition" /> |
 | <PyObject object="MultiAssetSensorDefinition"  />          | A special sensor definition class for multi asset sensors. You almost never want to use initialize this class directly. Instead, you should use the <PyObject object="multi_asset_sensor" decorator /> which returns a <PyObject object="MultiAssetSensorDefinition"  />                                                                                                |
 | <PyObject object="MultiSensorEvaluationContext"  />        | The context object passed to a multi asset sensor evaluation function. Has methods for fetching materialization events for assets                                                                                                                                                                                                                                       |
```

Comment:
I know this preexisted this PR (probably my fault) but probably worth fixing this while we're here.
`MultiSensorEvaluationContext` -> `MultiAssetSensorEvaluationContext`